models/classes/runner/time/storageFormat/QtiTimeStoragePackedFormat.php

```diff
@@ -273,24 +274,42 @@ public function decode($data)
                 }
 
                 foreach ($decodedData as $key => &$value) {
```

Comment:
Again, why do we need to go through all these values if we know what we are looking for

Reply:
Updated